examples/etcd/resdefs/bootstrap-flow.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: appcontroller.k8s/v1alpha1
+kind: Flow
```

Comment:
Are flows stored in etcd? I thought that they were only "virtual resources" without new adding new TPR.